geode-core/src/distributedTest/java/org/apache/geode/cache30/DistributedAckRegionCCEDUnitTest.java

```diff
@@ -394,22 +388,16 @@ public void testAggressiveTombstoneReaping() {
       }
       // now simulate a low free-memory condition
       TombstoneService.FORCE_GC_MEMORY_EVENTS = true;
-      WaitCriterion waitForGC = new WaitCriterion() {
-        public boolean done() {
-          return CCRegion.getCachePerfStats().getTombstoneGCCount() > initialCount;
-        }
+      Awaitility.waitAtMost(20, TimeUnit.SECONDS).pollInterval(1, TimeUnit.SECONDS)
+          .until(() -> CCRegion.getCachePerfStats().getTombstoneGCCount() > initialCount);
 
-        public String description() {
-          return "waiting for GC to occur";
-        }
-      };
-      Wait.waitForCriterion(waitForGC, 20000, 1000, true);
-      Wait.pause(5000);
+      Thread.sleep(5000);
```

Comment:
It looks like this test is making sure we don't do more than 20 tombstoneGCs in 5 seconds.
We really don't want hard coded sleeps in our tests so this test is worthy of its own ticket that it should be refactored to not sleep. Hopefully a better way exists to verify that the EXPIRED_TOMBSTONE_LIMIT has the desired effect.

Reply:
I agree, that all sleeps should go, what do you think should be done in the mean time?